civihr_default_theme/template.php

```diff
@@ -449,6 +449,9 @@ function _is_hrreports_current_path() {
  */
 function civihr_default_theme_js_alter(&$javascript) {
 
+  $path = drupal_get_path('theme',$GLOBALS['theme']);;
```

Comment:
There are two semicolons at the end of the line. 

I'm not sure why you're using `$GLOBALS['theme']` here, do you want the path to the active theme or the path to 'civihr_default_theme'?

Reply:
Changed to `drupal_get_path('theme', 'civihr_default_theme');`